src/pages/clouddocs/customizing.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Customizing"
+excerpt: "Customizing."
+categories: How To
+slug: customizing
+toc: true
+---
+<!-- ---
+
+copyright:
+  years: 2011, 2019
+lastupdated: "2019-04-12"
+
+keywords: annotator clinical data, clinical data, annotation
+
+subcollection: wh-acd
+
+--- -->
+
+<!-- # Customizing -->
+
+The Annotator for Clinical Data Configuration Editor supports extensive customization of the annotators as well as the ability to preview the customizations and export them in the form of a cartridge (zip file) that can be deployed directly to the service via the `/cartridges` APIs.
+
+The configuration editor facilitates the following customizations:
+
+| Customization | Description |
+|----|----|
+| Dictionaries | A set of terms describing a unique concept that is matched against the provided text to be analyzed. |
+| Derived Concepts | Rules for deriving concepts when one or more other concepts or tokens appears in the surrounding context. |
+| Filters | Conditional rules for omitting undesired annotations from the service response. |
+| Clinical Attributes | Configurable annotations built upon one or more other annotations. Optionally, semantically linked values in the surrounding context can be captured and associated with the attribute. |
+| Derived Clincal Attributes | Conditional logic and expressions for deriving new attributes based on values associated with other attributes. |
+| Attribute Qualifiers | Configurations for detecting qualifying terms in the immediate context of an attribute and capturing the qualifiers as a field within the output attribute annotation. |
+| Ontological Relations | Ontology configurations for extracting ontological relations between concepts cooccurring within the same sentence. |
+| CPT Code Mapping | Mapping configurations for outputting CPT codes from the concept and procedure annotators. |
+`Table 1. Customizations facilitated by the configuaration editor`
+
+[Contact your IBM representative](https://www.ibm.com/account/reg/us-en/signup?formid=MAIL-watsonhealthna) to learn more about leveraging the configuration editor to customize the behavior of the service.
+
+## Cartridge Deployment
+
+1. The consumer uses the {{site.data.keyword.wh-acd_short}} Configuration Editor to create a new cartridge (or modify an existing one) and customizes the contents (artifacts) of the cartridge to their domain. After that, the consumer will **Export** the cartridge in order to save a snapshot of the cartridge.
```

Comment:
"catridge" is misspelled in several places.

Reply:
{{site.data.keyword.wh-acd_short}} should be replaced with IBM Clinical Data Annotator